[AGM-2658] Update integrated-mode.md (AG for Android) (#303)
AdguardTeam/KnowledgeBaseVPN: docs/adguard-vpn-for-android/solving-problems/integrated-mode.md

sidebar_position: 3
---

AdGuard VPN has the VPN operating mode enabled by default, which uses its own [AdGuard VPN protocol](/general/adguard-vpn-protocol). It provides the best combination of connection speed and security. However, this operating mode does not allow AdGuard VPN and AdGuard Ad Blocker to work simultaneously. The Integrated mode, in turn, uses a special internal protocol, which makes it possible for the two apps to work together.

### How to enable Integrated mode

> If you already have AdGuard Ad Blocker when installing AdGuard VPN, Integrated mode will turn on automatically, thus allowing you to use our applications at the same time.

Reviewer: ```suggestion
If you have AdGuard Ad Blocker, Integrated Mode will turn on automatically when you install AdGuard VPN, allowing you to use both apps at the same time.
```

> If you are installing AdGuard Ad Blocker after AdGuard VPN — follow these steps to make using them together possible:

Reviewer: ```suggestion
If you are installing AdGuard Ad Blocker after AdGuard VPN, follow these steps to use them together:
```

> 1. Open AdGuard VPN for Android and tap the gear icon at the bottom right of the screen.

Reviewer: ```suggestion
1. Open AdGuard VPN for Android and tap the gear icon in the bottom-right corner of the screen.
```
